### DISCUSSION TITLE
License does not work

### DISCUSSION THREAD

**Discussion by andyshiller** [aclap-dev/video-downloadhelper#931] [License & Purchase requests]
### What's your issue?

Hello.
I purchased a Video DownloadHelper license from the Chrome web Store (Lifetime Premium VDH) in 2018. Previously, the app worked after logging into a Google account in Chrome. After updating the program, I cannot activate the license, as it now asks for a license key.

It is also impossible to restore the license. In response, a message arrives that such an e-mail address is not registered. 

The email address for which I bought the license: demed24@gmail.com . I still have a confirmation email and a Google order number.

### The 10 first digits (only 10) of your license number or email address

demed24@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
hi,
sorry i did not find your license. Please send a proof of purchase at licenses@downloadhelper.net
